example/trace/http/client/client.go

```diff
@@ -58,19 +57,29 @@ func main() {
 	tr := otel.Tracer("cloudtrace/example/client")
 
 	client := http.Client{Transport: otelhttp.NewTransport(http.DefaultTransport)}
-	ctx := baggage.ContextWithValues(context.Background(),
-		attribute.String("username", "donuts"),
-	)
-
+	b, err := baggage.New()
+	if err != nil {
+		panic(err)
+	}
+	member, err := baggage.NewMember("username", "donuts")
+	if err != nil {
+		panic(err)
+	}
+	b, err = b.SetMember(member)
+	if err != nil {
+		panic(err)
+	}
+	ctx := baggage.ContextWithBaggage(context.Background(), b)
```

Comment:
Slightly more concise:

```suggestion
	member, err := baggage.NewMember("username", "donuts")
	if err != nil {
		panic(err)
	}
	b, err := baggage.New(member)
	if err != nil {
		panic(err)
	}
	ctx := baggage.ContextWithBaggage(context.Background(), b)
```